### PR TITLE
Added OUYA controller

### PR DESCRIPTION
**Pull request by Quihico**

### CODE REVIEW

File: addons/game.controller.ouya/addon.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    </requires>
+    <extension point="kodi.game.controller" library="resources/layout.xml"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">OUYA controller</summary>
+        <description lang="en_GB">The OUYA is a crowd-funded Android-based microconsole released in 2013. The OUYA controller is a typical wireless gamepad with dual analogue sticks, a directional pad, 4 face buttons and pairs of back bumpers and triggers. It also includes a single-touch touchpad in the center of the controller.</description>
+        <disclaimer lang="en_GB"></disclaimer>
+        <platform>all</platform>
+        <assets>
+            <icon>resources/icon.png</icon>
+            <fanart>resources/icon.png</fanart>
```
Comment (Rechi) [inline addons/game.controller.ouya/addon.xml]: remove or submit a real fanart

Comment (Quihico) [reply addons/game.controller.ouya/addon.xml]: sorry about that, added fanart and corrected tag.

**Comment by garbear** [comment]
PR looks good. Two questions:

Can you minimize PNGs with a png minimizer tool? (I use pngcrush -brute)
Can you shorted the description in addon.xml? Long paragraphs are hard to read on a TV.

I would change:

The OUYA is a crowd-funded Android-based microconsole released in 2013. The OUYA controller is a typical wireless gamepad with dual analogue sticks, a directional pad, 4 face buttons and pairs of back bumpers and triggers. It also includes a single-touch touchpad in the center of the controller. 

to:

The OUYA is a crowd-funded Android-based microconsole released in 2013. The OUYA controller includes a single-touch touchpad in the center of the controller.</description>

Aim for something of that length but it's up to you.

Thanks for your contribution!

**Comment by Quihico** [comment]
Thanks for the suggestions, garbear.
Minimized all the png's and shortened the description in the addon.xml

**Comment by garbear** [comment]
awesome. can you squash the commits (`git rebase -i HEAD~3`) so the unminimized pngs don't end up in git history?

**Comment by Quihico** [comment]
Commits squashed.
Thanks again for the suggestions and accompanying instructions.

**Comment by garbear** [comment]
Can you PR to https://github.com/xbmc/repo-resources/tree/leia?

**Comment by garbear** [comment]
@Quihico where did you get this image? It would be awesome if we have profiles for the modern (ps3, ps4, xbox one, wii pro) controllers

**Pull request merged**